src/main/java/me/totalfreedom/totalfreedommod/util/History.java

```diff
@@ -0,0 +1,98 @@
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.UUID;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import me.totalfreedom.totalfreedommod.TotalFreedomMod;
+
+public class History
+{
+    public static void reportHistory(final CommandSender sender, final String username)
+    {
+        new BukkitRunnable() 
+        {
+            @Override
+            public void run()
+            {
+                UUID uuid;
+                FHistory history = null;
+                uuid = UUIDFetcher.fetch(username);
+                if (uuid != null)
+                {
+                    Gson gson = new GsonBuilder().create();
+                    String compactUuid = uuid.toString().replace("-", "");
+                    try
+                    {
+                        URL url = new URL("https://api.mojang.com/user/profiles/" + compactUuid + "/names");
+                        HttpURLConnection conn = (HttpURLConnection) url.openConnection();
+                        BufferedReader reader = new BufferedReader(new InputStreamReader(conn.getInputStream()));  
+                        FName[] oldNames = gson.fromJson(reader, FName[].class);
+                        reader.close();
+                        conn.disconnect();
+                        Arrays.sort(oldNames);
+                        history = new FHistory(uuid, oldNames);
+                        printHistory(sender, oldNames);
+                    }
+                    catch (Exception ex)
+                    {
+                        FLog.severe(ex);
+                    }
+
+                    if (history == null)
+                    {
+                        sender.sendMessage(ChatColor.RED + "Player not found!");
+                    }
+
+                }
+            }
+        }.runTaskAsynchronously(TotalFreedomMod.plugin());
+    }
+
+    private static void printHistory(CommandSender sender, FName[] oldNames)  
+    {
+        if (oldNames.length == 1)
+        {
+            sender.sendMessage(ChatColor.GREEN + oldNames[0].getName() + ChatColor.GOLD + " has never changed their name.");
+        }
+        else
+        {
+            sender.sendMessage(ChatColor.GOLD + "Original name: " + ChatColor.GREEN + oldNames[0].getName());
+
+            for (int i = 1; i < oldNames.length; i++)
+            {
+                DateFormat df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+                Date date = new Date(oldNames[i].changedToAt);
+                String formattedDate = df.format(date);
+                sender.sendMessage(ChatColor.BLUE + formattedDate + ChatColor.GOLD + " changed to " + ChatColor.GREEN + oldNames[i].getName());
+            }
+        }
+    }
+}
+
+class FName implements Comparable<FName>
```

Comment:
You can have more then two classes in one file. MainConfig does this for the ConfigDefaults class.